grants_tagger/models/mesh_xlinear.py

```diff
@@ -138,7 +146,11 @@ def save(self, model_path):
         self.xlinear_model_.save(model_path)
 
     def load(self, model_path, is_predict_only=True):
+        params_path = os.path.join(model_path, "params.json")
```

Comment:
so params were not saved and loaded using their save / load function? Did you raise an issue with them as to why? this might be a bug that need to be aware of or that you can fix on their side

Reply:
I think their parameters are saved fine, it's just the parameters of our class, and since we need some of the parameters for predictions, we needed to save it too.